src/api/Tidy5eSheetsApi.ts

```diff
@@ -665,6 +666,8 @@ export class Tidy5eSheetsApi {
     VehicleSheetRuntime.registerTab(registeredTab);
   }
 
+  spellSchool = SpellSchool;
```

Comment:
Consider making it dnd5e-like by creating a top-level API prop `config` to house things like this.
Or, possibly determine a similar organization scheme which keeps the number of props at the top level of the API from growing too much more.